app/fissile.go

```diff
@@ -799,11 +839,6 @@ roleLoop:
 			}
 
 		case model.RoleTypeBosh:
-
-			if errs := validation.ValidateRoleRun(role.Run); len(errs) != 0 {
```

Comment:
The code as is was moved to be called by `LoadManifest`. IOW, all that was done here is now done earlier, and for all commands. I am not sure of what variables you speak of here.